docs/reference/search/search-your-data/search-template.asciidoc

```diff
@@ -17,13 +27,14 @@ your searches without modifying your app's code.
 To create or update a search template, use the <<create-stored-script-api,create
 stored script API>>.
 
+
 The request's `source` supports the same parameters as the
 <<search-search-api-request-body,search API>>'s request body. `source` also
-supports https://mustache.github.io/[Mustache] variables, typically enclosed in
-double curly brackets: `{{my-var}}`. When you run a templated search, {es}
-replaces these variables with values from `params`.
+supports https://mustache.github.io/[Mustache] variables and has a https://github.com/elastic/elasticsearch/blob/main/modules/lang-mustache/build.gradle/[dependency] on the open source project https://github.com/spullara/mustache.java[mustache.java].
```

Comment:
Mentioning the gradle dependency assumes that users are familiar with Gradle, which is not necessarily the case, even for developers. I could see this being useful if the gradle dependency needed to be manually provided, but given that it's pre-installed and part of the default distribution, I find this mention redundant and potentially confusing. Fine specifying that we rely on an open source project and linking to it, but I would remove the gradle mention.